recipes/netpbm/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "netpbm" %}
+{% set version = "10.73.28" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/netpbm/files/super_stable/10.73.28/netpbm-10.73.28.tgz
```

Using variables in the URL makes it easier to update correctly:
```suggestion
  url: https://sourceforge.net/projects/netpbm/files/super_stable/{{ version }}/netpbm-{{ version }}.tgz
```